languages/julia/exercises/concept/annalyns-infiltration2/.docs/instructions.md

```diff
@@ -0,0 +1,100 @@
+In this exercise, you'll be extending the quest logic for the new RPG game a friend is developing.
+To make the game more interesting, your friend wants you to add random events that change the situation.
+
+The general setting hasn't changed from the previous exercise.
+The new random events are highlighted in the text below.
+
+The game's main character is Annalyn, a brave girl with a fierce and loyal pet dog.
+Unfortunately, disaster strikes, as her best friend was kidnapped while searching for berries in the forest.
+Annalyn will try to find and free her best friend, optionally taking her dog with her on this quest.
```

Comment:
I initially thought her dog was her best friend.

Reply:
Honestly, I'm a bit disappointed in Annalyn's judgement.

Reply:
I agree. I'll change it to "her best human friend"

Reply:
```suggestion
The game's main character is Annalyn, a brave girl with a fierce and loyal pet dog.
Unfortunately, disaster strikes, as her best human friend was kidnapped while searching for berries in the forest.
Annalyn will try to find and free her best friend, optionally taking her dog with her on this quest.
```